libc/src/string/memory_utils/utils.h

```diff
@@ -374,6 +374,12 @@ template <size_t SIZE> struct AlignHelper {
   uintptr_t offset_;
 };
 
+LIBC_INLINE void prefetch_for_write(CPtr dst) { __builtin_prefetch(dst, 1, 3); }
```

Comment:
Can you add inline comments to like so:
```
__builtin_prefetch(dst, /*write*/ 1, /*max locality*/ 3);
```
same below.

Reply:
Inline comments added